test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1904,7 +1904,7 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
         It "Image Parsing" {
             $dosUri = Get-WebListenerUrl -Test 'Dos' -query @{
                 dosType='img'
-                dosLength='5000'
```

Comment:
Discuss offline

Reply:
Delete the test, the old regex is no longer pathological due to changes in .NET.  Verified with .NET that they did optimizations.